ArkEcosystem/laravel-ui
# refactor: nodem navbar related adjustments

## Summary

Related to: https://app.clickup.com/t/p5fg09

- Accept icon and custom attributes on the navbar (so I can add a target="_blank" and the external icon)
![image](https://user-images.githubusercontent.com/17262776/125998282-5cb360f9-a352-4990-b1dd-26440759f006.png)
- Updates margins, padding, and separators
- This PR also affect msq that will contain another PR to fix the issues
- Also tested `ark.io` site and explorer and doesn't seem to be affected by this PR, still, you can double-check.

### To test:

Merge this on msq on nodem PRs and compare with the design see the related PRs for details:
1. Nodem: https://github.com/ArkEcosystem/nodem/pull/116
2. Msq: https://github.com/ArkEcosystem/marketsquare.io/pull/2180

## Checklist
